ow_lander/src/ow_lander/actions.py

```diff
@@ -72,6 +73,54 @@ def _compute_workspace_shoulder_yaw(x, y):
     yaw += math.asin(l / h)
     _assert_shou_yaw_in_range(yaw)
     return yaw
+    
+class _arm_faults_internal_messages_publish:
+    """Publish the arm failure messages to rostopic 'arm_faults_internal'"""
+    def __init__(self, err):
+      self.err = err
+      self.arm_faults_internal_pub = rospy.Publisher('arm_faults_internal', ArmFaultsStatus, queue_size = 10, latch = True)
```

Comment:
As a member of `ArmActionMixin` we could simply create this publisher once. There is no need to instantiate it everytime we need to publish a message.